node/service/src/chain_spec.rs

```diff
@@ -1521,6 +1521,107 @@ pub fn rococo_testnet_genesis(
 	}
 }
 
+
+/// Helper function to create rococo GenesisConfig for testing
+pub fn mousetrap_testnet_genesis(
```

Comment:
Why do we need a new runtime for this?

We already got the test-runtime that should be used for this kind of stuff.

Reply:
it would modify the behaviour significantly for all other tests as well and this is still early on exploration phase. Again, this is a DRAFT and tagged WIP.

Reply:
Yeah, I just wanted to warn you.

> behaviour significantly for all other tests as well

If you mean by this that you changed constants in the runtime. Exactly for this reason the constants in the test-runtime are read from the storage, to make them modifiable.

Reply:
That's a good point! Thanks